Error when building with CRA

I've updated to version `10.0.0` and have this error on build with `react-scripts`

```
./node_modules/react-youtube/dist/YouTube.mjs
Can't import the named export 'jsx' from non EcmaScript module (only default export is available)
```

I haven't changed anything from when using version `9.0` and seeing changelog, nothing is supposed to have changed 🤔 

ps: Don't think it's related, but i'm obviously using `react@18`

+1 
My full error message is:
```
Compiled with problems:

ERROR in ./node_modules/react-youtube/dist/YouTube.mjs 6:0-40

Module not found: Error: Can't resolve 'react/jsx-runtime' in 'C:\Users\XXXXX\XXX\node_modules\react-youtube\dist'
Did you mean 'jsx-runtime.js'?
BREAKING CHANGE: The request 'react/jsx-runtime' failed to resolve only because it was resolved as fully specified
(probably because the origin is strict EcmaScript Module, e. g. a module with javascript mimetype, a '*.mjs' file, or a '*.js' file where the package.json contains '"type": "module"').
The extension in the request is mandatory for it to be fully specified.
Add the extension to the request.
```
react: 16.14.0
react-scripts: 5.0.1
react-youtube: 10.0.0

--- a/packages/react-youtube/package.json
+++ b/packages/react-youtube/package.json
@@ -3,7 +3,7 @@
   "version": "1.0.0",
   "description": "React.js powered YouTube player component",
   "main": "dist/YouTube.js",
-  "module": "dist/YouTube.mjs",
+  "module": "dist/YouTube.esm.js",
   "types": "dist/YouTube.d.ts",
   "sideEffects": false,
   "files": [
@@ -45,7 +45,7 @@
     "semantic-release": "19.0.5",
     "semantic-release-monorepo": "7.0.5",
     "ts-jest": "29.0.0",
-    "tsup": "6.2.3"
+    "tsup": "6.5.0"
   },
   "peerDependencies": {
     "react": ">=0.14.1"

--- a/packages/react-youtube/src/YouTube.tsx
+++ b/packages/react-youtube/src/YouTube.tsx
@@ -1,3 +1,4 @@
+/** @jsxRuntime classic */
 import PropTypes from 'prop-types';
 import React from 'react';
 import isEqual from 'fast-deep-equal';

--- a/packages/react-youtube/tsup.config.ts
+++ b/packages/react-youtube/tsup.config.ts
@@ -1,0 +1,11 @@
+import { defineConfig } from 'tsup';
+
+export default defineConfig({
+  sourcemap: true,
+  clean: true,
+  outExtension({ format }) {
+    return {
+      js: format !== 'cjs' ? `.${format}.js` : '.js',
+    };
+  },
+});

